src/modules/video_player/index.js

```diff
@@ -94,6 +104,36 @@ class VideoPlayerModule {
             $(VIDEO_PLAYER_SELECTOR).on('click', '.player-overlay.player-fullscreen-overlay', handlePlayerClick);
         }
     }
+
+    hideMouseFullscreen() {
+        const showMouse = () => {
+            $('.video-player--fullscreen')
+                .toggleClass('bttv-hide-cursor-fullscreen', false);
+        };
+
+        const hideMouse = () => {
+            $('.video-player--fullscreen')
+                .toggleClass('bttv-hide-cursor-fullscreen', true);
+        };
+
+        $('body').off('mousemove', '.video-player--fullscreen');
+        clearTimeout(hideMouseTimer);
+        showMouse();
+
+        if (settings.get('hideMouseFullscreen') === true) {
+            hideMouseTimer = setInterval(() => {
+                hideMouse();
+            }, 3000);
+
+            $('body').on('mousemove', '.video-player--fullscreen', () => {
+                showMouse();
+                clearTimeout(hideMouseTimer);
```

Comment:
we should not create a timer that runs to hide the mouse, that is just wrong. you can easily use a debounce function that gets called on mouse move that hides the mouse after a timeout.

Reply:
I did try that, but an issue arises if a user goes fullscreen without moving their mouse. If they use a hotkey for instance. Then they would have to move their mouse initially for the timer to start.

One solution to this I was thinking of, is adding an event in watcher.js that fires when the screen goes fullscreen, then in the video_player module starting the timer off that.

Reply:
i think requiring mouse movement is an acceptable constraint to hiding the mouse. it's not worth adding additional plumbing for an edge case benefitting only a few people.

Reply:
Alright, then that should simplify the code a bit.